olive/passes/pytorch/gptq.py

```diff
@@ -41,7 +42,7 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
             ),
             "layers_block_name": PassConfigParam(
                 type_=str,
-                default_value="model.layers",
+                default_value=None,
                 description=(
                     "Block name to quantize. Default value is model.layers. "
```

Comment:
oh, btw this still refers to a default value.

Reply:
If I leave that as default value then line 181 will always succeed making the else part non-reachable code.

Reply:
sorry, I meant the description still refers to a default value when there is no default value anymore.